### PR TITLE
Add new page (epic DC character)

### PR DESCRIPTION
**Pull request by TDbouk**

### CODE REVIEW

File: english/DC-Comics/epic_DC_Character.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Epic DC
```
Comment (casey-collab) [inline english/DC-Comics/epic_DC_Character.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by TDbouk** [comment]
Pleas review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by TDbouk** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :purple_heart:

**Pull request merged**